src/src/fileLoger/hillel/FileLogger.java

```diff
@@ -0,0 +1,53 @@
+package fileLoger.hillel;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class FileLogger {
```

Comment:
It seems, that you haven't completed some points from the task, so the mark will be lowered for that as well.